### DISCUSSION TITLE
password reset

### DISCUSSION THREAD

**Discussion by RubyEyedReaper** [wizarrrr/wizarr#398] [Feature Requests]
ability to reset user passwords for jelly so admin doesn't need to manually change them

**Comment by mattague** [comment]
Yes, something similar to how jfa-go works would be awesome.

**Reply by Sandoiitchisan** [reply]
I definitely upvote this. jfa-go is as awesome as it's ugly, and Wizarr is as awesome as it misses features !

**Comment by xaralec** [comment]
Ideally, the Jellyfin "Forgot Password" button should send a password reset link directly via email using SMTP if an email is linked to the account.

If no email is associated with the account, then it should fall back to the current PIN code method.
Example: A tool like Wizarr can monitor Jellyfin directories, and when a PIN code is generated, it sends a password reset link by email if the account has an email linked.

It would also be useful, through a plugin or CSS customization, to modify the message displayed when clicking "Forgot Password" so it reflects this change.
